jekyll/_cci2/google-auth.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: classic-docs
+title: Authenticating Google Cloud Platform
+description: Authenticating Google Cloud Platform
+---
+
```

Comment:
add: [Deploy]({{ site.baseurl }}/2.0/deployment_integrations/) > Authenticating Google Cloud Platform 

this is the navigation breadcrumb we should add for all new articles (changing the landing page link to wherever the top-level page lives